cron/monthly/email-report.js

```diff
@@ -34,7 +34,10 @@ const init = () => {
           'name',
           'currency',
           'tags'
-      ]
+      ],
+      where: {
+        slug: 'railsgirlsatl'
```

Comment:
Did you mean to check this in? Won't it require another PR once you want to take it live?